packages/firebase_core/firebase_core_web/lib/firebase_core_web.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:js' as js;
+import 'dart:js_util' as js_util;
```

Comment:
You can `import package:js/js_util.dart` instead I think.

Reply:
(Check package:js out [here](https://github.com/dart-lang/sdk/tree/master/pkg/js/lib))

Reply:
Done.